README.md

```diff
@@ -9,6 +9,8 @@ Legofy is a python program that takes a static image or gif and makes it so that
 </a>
 <img alt="After" title="After" height="500" src="legofy/assets/flower_lego.png?raw=true">
 
+## Contributing
```

Comment:
Usually contributing is near the end of the Readme. Why use a second tier header when all the other sections use third tiers?